spec/unit/type/file/selinux_spec.rb

```diff
@@ -50,13 +50,17 @@
     end
 
     it "should handle no default gracefully" do
-      expect(@sel).to receive(:get_selinux_default_context).with(@path, :file).and_return(nil)
+      skip if Puppet::Util::Platform.windows?
+      expect(@sel).to receive(:get_selinux_default_context_with_handle).with(@path, nil).and_return(nil)
       expect(@sel.default).to be_nil
     end
 
-    it "should be able to detect matchpathcon defaults" do
+    it "should be able to detect default context" do
+      skip if Puppet::Util::Platform.windows?
```

Comment:
It'd be good to test what the behavior is on Windows, so something like:

```
 it "should be able to detect default context on platforms other than Windows", unless: Puppet::Util::Platform.windows?
  # existing test
end
```

And then add a windows test:

```
 it "returns a nil default context on Windows", if: Puppet::Util::Platform.windows? do
  ... whatever stubbing is required ...
  expect(@sel.default).to be_nil
end
```

That way we can verify the code returns the expected value on Windows (as opposed to getting a `NoMethodError` or worse)